DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/Commands/NoteCommands.cs

```diff
@@ -18,5 +20,22 @@ public NoteCommands(NoteContext notesContext)
             _notesContext.Notes.Add(noteEntity);
             await _notesContext.SaveChangesAsync();
         }
+
+        public async Task<bool> DeleteNote(string userId, string collaboratorId)
+        {
+            var note = _notesContext.Notes.SingleOrDefault(n => n.UserId == userId 
```

Comment:
I have added unique constraint for userId and collaboratorId, so I can prevent using SingleOrDefault. Fix commited